deployer/aws-lambda/README.md

```diff
@@ -54,3 +54,48 @@ yarn make-package
 
 and if the deployment package is different from what is already in AWS,
 it will upload and publish a new version.
+
+## Debugging the `content-origin-request` handler
+
+You can simulate what Lambda@Edge does, but on your laptop.
+To start it, run:
+
+```sh
+cd aws-lambda
+cd content-origin-request
+yarn install
+yarn serve
+```
+
+This will start a server at <http://localhost:7000>. It's meant to work as similar
+as it can to code when run within Lambda@Edge. To test it, try:
```

Comment:
```suggestion
This will start a server at <http://localhost:7000>. It's meant to work much
the same as when our Lambda@Edge function is run within AWS. To test it, try:
```